packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/index.js

```diff
@@ -45,25 +41,22 @@ import {
   VisuallyHidden,
 } from '@strapi/design-system';
 import { Plus, Pencil, Trash, EmptyDocuments } from '@strapi/icons';
-import reducer, { initialState } from './reducer';
 import adminPermissions from '../../../../../permissions';
 
 const ListView = () => {
-  const {
-    isLoading,
-    allowedActions: { canCreate, canRead, canUpdate, canDelete },
-  } = useRBAC(adminPermissions.settings.webhooks);
-  const toggleNotification = useNotification();
-  const isMounted = useRef(true);
-  const { formatMessage } = useIntl();
+  const [webhooks, setWebhooks] = useState([]);
+  const [webhookToDelete, setWebhookToDelete] = useState(null);
+  const [webhooksToDelete, setWebhooksToDelete] = useState([]);
   const [showModal, setShowModal] = useState(false);
-  const [{ webhooks, webhooksToDelete, webhookToDelete, loadingWebhooks }, dispatch] = useReducer(
-    reducer,
-    initialState
-  );
-  const { notifyStatus } = useNotifyAT();
 
+  const { formatMessage } = useIntl();
+  const toggleNotification = useNotification();
+  const {
+    isLoading: RBACLoading,
```

Comment:
can we call this `isRBACLoading` please? That's kindof a convention we use everywhere.